lab-carolina/webpack.config.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const HTMLPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: `${__dirname}/app/entry.js`,
+  output: {
+    filename: 'bundle.js',
+    path: 'build',
```

Comment:
I believe this is incorrect syntax, resulting in an error on build, with webpack 2.x. The property of path needs to be an absolute path.